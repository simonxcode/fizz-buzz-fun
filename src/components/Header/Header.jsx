import React from 'react'

const Header = () => (
  <div className="flex justify-center items-center text-4xl text-center font-bold p-4 drop-shadow-xl">
    Fizz Buzz Fun
  </div>
)

export default Header
