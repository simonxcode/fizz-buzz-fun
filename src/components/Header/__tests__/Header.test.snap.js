import React from 'react'
import renderer from 'react-test-renderer'
import Header from '../Header'

it('Header component should match snapshot', () => {
  const tree = renderer.create( <Header />).toJSON()
  expect(tree).toMatchSnapshot()
});
