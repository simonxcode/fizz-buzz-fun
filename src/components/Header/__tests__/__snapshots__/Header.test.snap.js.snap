// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match snapshot 1`] = `
<div
  className="flex justify-center items-center text-4xl text-center font-semibold p-4 text-emerald-900 drop-shadow-xl"
>
  Fizz Buzz Fun
</div>
`;
